charts/kasperleyn/README.md

```diff
@@ -0,0 +1,36 @@
+kasperleyn
+==========
+A Helm 2 chart to deploy Caasperli
+
+Current chart version is `0.3.1`
+
+Source code can be found [here](https://github.com/adfinis-sygroup/potz-holzoepfel-und-zipfelchape)
+
+
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"Always"` |  |
+| image.repository | string | `"adfinissygroup/potz-holzoepfel-und-zipfelchape"` |  |
+| image.tag | string | `"latest"` |  |
```

Comment:
To be consistent with the Helm 3 version of the chart we should remove the default to `latest` here.

Reply:
Helm 3 and Helm 2 charts aren't consistent, that why I have them both.